.github/workflows/ci-e2e-badger.yaml

```diff
@@ -26,6 +26,10 @@ jobs:
 
     - uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938 # v4.2.0
 
+    - name: Fetch git tags
+      run: |
+        git fetch --prune --unshallow --tags
```

Comment:
did you check if we can achieve the same result simply by massing an extra flag to actions/checkout above?